### PR TITLE
Fix Favorites test cases

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://rackspace.atlassian.net/browse/MNRVA-943

 **Description:**
Fixed favorites test cases

**Testing:**

`npm run test:unit`

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: e2e/src/features/misc/spinnertest.e2e-spec.ts [hunk 1]
```diff
@@ -36,14 +36,12 @@ describe("Test page loading/spinner functionality",()=>{
           var theLoader = await browser.executeScript("return document.querySelectorAll('.gbl-spinner-show').length"
           );
           expect(theLoader).toEqual(1);
-          // console.log("result is:" + JSON.stringify(theLoader));
```
Comment (FarajiA) [inline e2e/src/features/misc/spinnertest.e2e-spec.ts:39]: We can delete `console.log` lines

File: src/app/_features/events/components/list/eventslist.component.ts [hunk 2]
```diff
@@ -58,7 +58,6 @@ export class EventslistComponent implements OnInit {
   triggerConfirm() {
     this.selectedEvents.map((value) => {
       this.eventService.deleteEvent(value).subscribe(data => {
-          //console.log("data ", data);
```
Comment (FarajiA) [inline src/app/_features/events/components/list/eventslist.component.ts:61]: Here also

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts [hunk 3]
```diff
@@ -221,11 +271,26 @@ export class VisualizePage {
 
   // Get list of metric on the basis of Group
   getlistOfMetric() {
+    /*let query = {
```
Comment (FarajiA) [inline src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts:274]: We can delete this comment block

**Review by FarajiA — APPROVED** [review]

File: src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts (hunk 3 above)
Comment (lovedeep92) [reply src/app/_features/visualize/pages/visualize.page/visualize.page.component.ts:274]: done!!

File: src/app/_features/events/components/list/eventslist.component.ts (hunk 2 above)
Comment (lovedeep92) [reply src/app/_features/events/components/list/eventslist.component.ts:61]: this is already deleted!

File: e2e/src/features/misc/spinnertest.e2e-spec.ts (hunk 1 above)
Comment (lovedeep92) [reply e2e/src/features/misc/spinnertest.e2e-spec.ts:39]: already deleted, that's why showing red line.

**Pull request merged**